flowdb/Dockerfile

```diff
@@ -12,7 +12,7 @@
 #  on the official Debian Stretch (9) image.
 #
 ARG POSTGRES_RELEASE=11
-ARG POSTGRES_DIGEST=235970ccf70d6e426e8da502ad64164b3a35396d44b464aacefaad902a31041a
+ARG POSTGRES_DIGEST=40b48cf04185e69c5a9183a986fcbbfd105e5991a70b1980e9a641b53ba5ead7
```

Comment:
Where do you get this digest from? How can we verify it is the one for 11.1?

Reply:
```bash
docker pull postgres:11.1
11.1: Pulling from library/postgres
Digest: sha256:40b48cf04185e69c5a9183a986fcbbfd105e5991a70b1980e9a641b53ba5ead7
Status: Downloaded newer image for postgres:11.1
```

```bash
docker pull postgres:11
11: Pulling from library/postgres
Digest: sha256:40b48cf04185e69c5a9183a986fcbbfd105e5991a70b1980e9a641b53ba5ead7
Status: Image is up to date for postgres:11
```